backend.native/tests/framework/values_generics/values.kt

```diff
@@ -0,0 +1,344 @@
+/*
```

Comment:
Maybe it makes sense to avoid copy-pasting `backend.native/tests/framework/values/values.kt` and instead do generic-specific tests.

Reply:
Updated in latest